### PR TITLE
Blog/lw5 pg json

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/www/_blog/2022-08-19-pg_jsonschema.mdx [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: JSON Schema support for JSON/B Columns
+description: '...'
```
Comment (kiwicopple) [inline apps/www/_blog/2022-08-19-pg_jsonschema.mdx:3]:
```suggestion
description: "Today we're releasing pg_jsonschema, a Postgres extension for JSON validation."
```

File: apps/www/_blog/2022-08-19-pg_jsonschema.mdx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+thumb: lw5-realtime/thumb.jpg
+tags:
+  - launch-week
+date: '2022-08-19'
+toc_depth: 3
+---
+
+# Supabase: JSON Schema support for JSON/B Columns
+
+## TL;DR
```
Comment (kiwicopple) [inline apps/www/_blog/2022-08-19-pg_jsonschema.mdx:15]:
```suggestion
## TLDR
```

Comment (kiwicopple) [inline apps/www/_blog/2022-08-19-pg_jsonschema.mdx:15]:
```suggestion
```

File: apps/www/_blog/2022-08-19-pg_jsonschema.mdx [hunk 3]
```diff
@@ -0,0 +1,209 @@
+---
+title: JSON Schema support for JSON/B Columns
```
Comment (kiwicopple) [inline apps/www/_blog/2022-08-19-pg_jsonschema.mdx:2]:
```suggestion
title: Supabase now support JSON Schema validation for JSON/B Columns
```

File: apps/www/_blog/2022-08-19-pg_jsonschema.mdx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+author: oli_rice
+image: lw5-realtime/thumb.jpg
+thumb: lw5-realtime/thumb.jpg
+tags:
+  - launch-week
+date: '2022-08-19'
+toc_depth: 3
+---
+
+# Supabase: JSON Schema support for JSON/B Columns
```
Comment (kiwicopple) [inline apps/www/_blog/2022-08-19-pg_jsonschema.mdx:13]:
```suggestion
```

**Review by w3b6x9 — APPROVED** [review]

File: apps/www/_blog/2022-08-19-pg_jsonschema.mdx [hunk 5]
```diff
@@ -0,0 +1,207 @@
+---
+title: Supabase now support JSON Schema validation for JSON/B Columns
+description: "Today we're releasing pg_jsonschema, a Postgres extension for JSON validation."
+author: oli_rice
+image: lw5-realtime/thumb.jpg
+thumb: lw5-realtime/thumb.jpg
```
Comment (w3b6x9) [inline apps/www/_blog/2022-08-19-pg_jsonschema.mdx:6]: @MildTomato got art work for this? cc/ @olirice

**Pull request merged**